docs/release-notes/charmcraft-4.1.rst

```diff
@@ -0,0 +1,95 @@
+:orphan:
+.. _release-4.1:
+
+Charmcraft 4.1 release notes
+============================
+
+*Unreleased*
+
+Learn about the new features, changes, and fixes introduced in Charmcraft 4.1.
+
+
+Requirements and compatibility
+------------------------------
+
+For development and testing, Charmcraft requires a host with a minimum of 4GB RAM
+running a Linux distribution compatible with systemd.
+
+All versions of Charmcraft require the following software:
+
+- systemd
+- `snapd`_
+- Either `LXD`_ or `Multipass`_
+
+We recommend you install the `Charmcraft snap <https://snapcraft.io/charmcraft>`_. It
+comes bundled with all its dependencies.
+
+Non-snap installations of Charmcraft have the following dependencies:
+
+- Python 3.10 or higher
+- libgit2 1.7
+- `skopeo`_
+- `Spread`_
+
+What's new
+----------
+
+Charmcraft 4.1 brings the following new features.
+
+Ubuntu 25.10 and 26.04
+~~~~~~~~~~~~~~~~~~~~~~
+
+Charmcraft 4.1 adds experimental support for Ubuntu 25.10 and Ubuntu 26.04 bases.
+As an interim base, the behaviour of Charmcraft when packing an Ubuntu 25.10 charm is
+not guaranteed to be stable, but is instead consistent with the evolving changes
```

Comment:
> consistent with the evolving changes

Might come across as obtuse. It could be clearer to say it's main purpose is for testing software and charms in preparation for 26.04.